src/styles/index.ts

```diff
@@ -2954,10 +2954,25 @@ const styles = (theme: ThemeColors) =>
                 color: theme.heading,
                 padding: 0,
                 lineHeight: undefined,
+                paddingHorizontal: 0,
+                paddingVertical: 0,
+                borderTopLeftRadius: 0,
+                borderBottomLeftRadius: 0,
+                borderTopRightRadius: 0,
+                borderBottomRightRadius: 0,
```

Comment:
Can we just use this instead?
```suggestion
                padding: 0,
                borderRadius: 0
```

Reply:
What's the regression that you mentioned?

Reply:
This is only remaining thing @ZhenjaHorbach